dask_ml/model_selection/_incremental.py

```diff
@@ -652,13 +665,17 @@ class IncrementalSearch(BaseIncrementalSearch):
         If ``patience`` is used the maximum number of ``partial_fit`` calls
         between ``score`` calls.
 
-    tol : float, default 0.001
+    tol : float, default -0.01
```

Comment:
Why flip the sign here? IIRC, these are typically expressed as positive numbers.

Reply:
Could you remind me what we decided here? I think that I was in favor of `tol` being positive by default, since that aligns with the default in scikit-learn.

Reply:
Regardless of the default, it'd be good to note that `tol<0` is well defined (we're allowed to stop on a model that does not too much worse that the one before it).

edit: Ah, good I see that you have that below.

Reply:
Whoops, I need to make the default `tol` positive. PyTorch's `ReduceLROnPlateau` also defaults to a postive `tol` (IIRC, I linked to the source elsewhere).

TODO: make tol positive again